src/main/java/fr/jmmc/oiexplorer/core/model/oi/SubsetFilter.java

```diff
@@ -215,6 +215,29 @@ public void toString(final StringBuilder sb, final boolean full) {
             sb.append('}');
         }
     }
+
+    /**
+     * returns a short String representation. used for logging for example
+     *
+     * @return a short string representation.
+     */
+    public String toShortString() {
+        final StringBuilder sb = new StringBuilder();
+        sb.append("(").append(targetUID == null ? "null-target" : targetUID);
+        sb.append(",").append(insModeUID == null ? "null-insmode" : insModeUID);
+        sb.append(",").append(nightID == null ? "null-night" : nightID);
+        if (tables == null) {
+            sb.append(",null-tables)");
+        } else {
+            sb.append(",[");
+            for (int i = 0, s = tables.size(); i < s; i++) {
+                sb.append(i == 0 ? "" : ",");
+                sb.append(tables.get(i).extName);
```

Comment:
not enough, missing file and extNb, see TableUID.toString !
Just add toShortString() too in TableUID to make a shorter string and call it here (object oriented) ?